src/org/usfirst/frc2876/Steamworks2017/Robot.java

```diff
@@ -51,6 +51,10 @@
 	// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
 
 	public static Vision vision;
+	
+	public static final boolean IS_TURN_PID_FUNCTIONAL = false;
+	public static final boolean IS_DISTANCE_PID_FUNCTIONAL = false;
+	public static final boolean IS_STRAIGHT_PID_FUNCTIONAL = false;

```

Comment:
What's the purpose of these 3 vars? I understand they have something to do with 3 pids in DriveTrain.. I'm asking why do we these 3 vars? How are they intended to be used? For what?

Why are they in Robot.java if they are all referring to something part of DriveTrain?